lib/rust/parser/generate-java/java/org/enso/syntax2/Parser.java

```diff
@@ -27,8 +27,7 @@ public final class Parser implements AutoCloseable {
     } catch (URISyntaxException | LinkageError e) {
       System.err.println("Cannot load " + parser);
       File root = new File(".").getAbsoluteFile();
-      if (!searchFromDirToTop(e, root, "target", "rust", "x86_64-apple-darwin", "debug", name)
-          && !searchFromDirToTop(e, root, "target", "rust", "debug", name)) {
+      if (!searchFromDirToTop(e, root, "target", "rust", "debug", name)) {
```

Comment:
Similarly as in the other case, those shouldn't be hardcoded.

Reply:
OK, this is in static initializer so it will be hard to configure. Note that the `debug` bit is probably gonna be problematic for non-dev releases